platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ProjectColorTabDecorator.java

```diff
@@ -161,8 +161,16 @@ public void paintAfter( TabData tab, Graphics g, Rectangle tabRect, boolean isSe
         }
         g.setColor( c );
         Rectangle rect = new Rectangle( tabRect );
-        rect.y += rect.height - 3;
-        rect.grow( -1, -1 );
+        int underlineHeight = UIManager.getInt("nb.multitabs.underlineHeight"); // NOI18N
```

Comment:
Looking at FlatLaf.properties ( https://github.com/apache/netbeans/blob/3139141059141caab4a6af1158cce7131e1b40f9/platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLaf.properties ), the definition is "nb.multitabs.underlineHeight={scaledInteger}3". What does "scaledInteger" mean? How does that get parsed here?

Reply:
`{scaledInteger}` means that FlatLaf scales this value (when initializing the Laf) using the FlatLaf user scale factor, which is used for Java 8 but also on Linux. When running in Java 9 the FlatLaf user scale factor is usually 1.

In other places I usually invoke method `UIScale.scale( intValue )`, but in this case it would require that Multi-tabs project depends on FlatLaf library project, which I wanted to avoid.

The downside of using `{scaledInteger}` is that it is not updated when the user changes scaling in Windows Settings.

Reply:
Ah, so I get it's FlatLAF that reads the property file and puts the scaled value into UIManager. Sounds good.

> The downside of using {scaledInteger} is that it is not updated when the user changes scaling in Windows Settings.
Presumably it's scaled via the default Graphics2D transform in that case. Sounds correct in any case.